libs/sdk-ui-ext/src/internal/components/configurationPanels/LineChartBasedConfigurationPanel.tsx

```diff
@@ -86,4 +103,20 @@ export default class LineChartBasedConfigurationPanel extends BaseChartConfigura
             </BubbleHoverTrigger>
         );
     }
+
+    private isContinuousLineSelectable(): boolean {
+        const { insight, properties, type } = this.props;
+        let isStackingMeasures = properties?.controls?.stackMeasures;
+        if (isStackingMeasures) {
+            return false;
+        } else if (typeof isStackingMeasures === "undefined" && isAreaChart(type)) {
+            isStackingMeasures =
+                insight.insight.buckets.find((item) => item.localIdentifier === BucketNames.MEASURES)?.items
```

Comment:
Please check, maybe this https://github.com/gooddata/gooddata-ui-sdk/blob/master/libs/sdk-ui-ext/src/internal/utils/bucketRules.ts#L67 or some other similar helper here could be handy.

Reply:
Ah, that's a pity, what about:
```
import { insightBucket } from "@gooddata/sdk-model";
import { BucketNames } from "@gooddata/sdk-ui";
...
        const measuresBucket = insightBucket(insight, BucketNames.MEASURES);
        const hasMoreThanOneMetric = measuresBucket.items.length > 1;
```
Please check if that could work, thanks.

Reply:
I have updated this. Thanks!